layouts/partials/tw/frontpage.html

```diff
@@ -119,7 +119,7 @@ <h2 class="text-2xl font-extrabold">
     <div
       data-ga-category="midpage"
       class="bottom-0 row-start-4 md:relative md:col-span-9 md:mx-0 md:overflow-hidden md:rounded xl:col-span-10"
-      :class="isSticky ? 'sticky -mx-4 bottom-0 md:mx-0' : 'relative mx-0 rounded overflow-hidden'"
+      :class="isSticky ? 'hidden sticky -mx-4 bottom-0 md:mx-0' : 'relative mx-0 rounded overflow-hidden'"
```

Comment:
We can just delete all this and change it to include newsletter/papost-hidden.html instead.